modules/node/src/test/utils/cfCore.ts

```diff
@@ -177,6 +177,7 @@ export const createStoredAppChallenge = (
     versionNumber: One,
     finalizesAt: Zero,
     status: StoredAppChallengeStatus.IN_DISPUTE,
+    chainId: 1339,
```

Comment:
These chainIds shouldn't be hardcoded, we have an `env.defaultChain` var to use instead